v23.1/backup-and-restore-overview.md

```diff
@@ -42,6 +42,13 @@ Backup / Restore  | Description  | Product Support
 - [Incremental backups with explicitly specified destinations](take-full-and-incremental-backups.html#incremental-backups-with-explicitly-specified-destinations)
 - [Exclude a table's data from backups](take-full-and-incremental-backups.html#exclude-a-tables-data-from-backups)
 
+## Backups with locality requirements
+
+CockroachDB supports two types of backup that use a node's locality to determine how a backup job runs or where the backup data is stored:
+
+- [Locality-restricted backup execution](take-locality-restricted-backups.html): Specify a set of locality filters for a backup job in order to restrict the nodes that can participate in the backup process. This ensures that the backup job is executed by nodes that meet certain requirements, such as being located in a specific region or having access to a certain storage bucket.
```

Comment:
Specify a set of locality filters for a backup job in order to restrict the nodes that can participate in the backup process..... to that locality